runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -308,6 +305,23 @@ else if (poisonVarp >= VENOM_VALUE_CUTOFF)
 
 			lastPvpVarb = pvpVarb;
 		}
+
+		if (lastImbuedHeartVarb != imbuedHeartCooldownVarb && config.showImbuedHeart())
+		{
+			if (imbuedHeartCooldownVarb == 0)
+			{
+				log.debug("The Imbued Heart has regained its magical power, clearing timer");
```

Comment:
These log statements don't really help clarify anything and can be removed. (there's already debug logging for removal of infoboxes via `InfoboxManager`)